extensions/pom.xml

```diff
@@ -98,6 +102,12 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
these changes mean that extensions/all-extensions/pom.xml can no longer have this pom as a parent.